CONTRIBUTING.md

```diff
@@ -116,14 +116,24 @@ dtls_get_peer(const dtls_context_t *ctx, const session_t *session) {
 ...
 }
 ```
+* If the function definition exceeds the line length of 80 characters caused by
+  the parameter-list, the split this list on multiple lines aligning with the 
+  first parameter.
 
+```
+int
+dtls_read(const dtls_context_t *ctx, const session_t *session,
+          uint8_t* buffer, size_t buffer_length) {
+...
+}
+```
 * Declarations in header files do not follow the previous rule. For
```

Comment:
This has now become ambiguous.  It refers to the return type being on the preceding line, not how you split long lines.

* Declarations in header files keep the return type on the same line as the function definition.  For

Reply:
Just to mention:
I'm not sure, which format should be applied for function declarations in *.c files.

Reply:
As these functions all should be prefixed by `static`, this increases the line length requirement.  My suggestion for the .c files is that the static + return type are kept on the previous line.